unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ExploreHUD/Scripts/BaseSceneCellView.cs

```diff
@@ -81,13 +81,9 @@ void IMapDataView.SetMinimapSceneInfo(MinimapMetadata.MinimapSceneInfo info)
 
         if (GetThumbnail() == null)
         {
-            string url = mapInfo.previewImageUrl;
-            if (string.IsNullOrEmpty(url))
-            {
-                url = MapUtils.GetMarketPlaceThumbnailUrl(mapInfo, 196, 143, 50);
-            }
-
-            FetchThumbnail(url);
+            FetchThumbnail(mapInfo.previewImageUrl,
+                onFetchFail: () => FetchThumbnail(MapUtils.GetMarketPlaceThumbnailUrl(mapInfo, 196, 143, 50),
```

Comment:
What these magic numbers are for? We can't use constants here?

Reply:
added constant for those numbers